### PR TITLE
Bump eslint to 7 and associated dependencies

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

File: src/components/Questions/AnswerInput.vue [hunk 1]
```diff
@@ -98,6 +98,7 @@ export default {
 			if (this.answer.local) {
 
 				// Dispatched for creation. Marked as synced
+				// eslint-disable-next-line vue/no-mutating-props
 				this.answer.local = false
```
Comment (skjnldsv) [inline src/components/Questions/AnswerInput.vue:102]: Fine for this use, this is a temporary edit and we're editing the object content

File: src/views/Create.vue [hunk 2]
```diff
@@ -91,7 +91,7 @@
 					:answer-type="answerTypes[question.type]"
 					:index="index + 1"
 					:max-string-lengths="maxStringLengths"
-					v-bind.sync="question"
+					v-bind="question"
```
Comment (skjnldsv) [inline src/views/Create.vue:94]: Apparently that had no effect :thinking:

File: src/components/Questions/AnswerInput.vue (hunk 1 above)
Comment (jotoeri) [reply src/components/Questions/AnswerInput.vue:102]: 🐘

File: src/views/Create.vue (hunk 2 above)
Comment (jotoeri) [reply src/views/Create.vue:94]: :D insteresting.... But functionality still works?

**Review by jotoeri — APPROVED** [review]
Hui, quite many... 👍 
I didn't test now, but i assume everything still works...?

File: src/views/Create.vue (hunk 2 above)
Comment (skjnldsv) [reply src/views/Create.vue:94]: https://vuejs.org/v2/guide/components-custom-events.html#sync-Modifier
Capture d’écran_2020-11-04_08-26-43

Comment (skjnldsv) [reply src/views/Create.vue:94]: What is weird is that it still works indeed

Comment (skjnldsv) [reply src/views/Create.vue:94]: So v-bind.sync is allowed, it's the v-bind within a v-for that was the issue :thinking:

Comment (skjnldsv) [reply src/views/Create.vue]: Fixed! That was weird! :grin:

**Comment by jotoeri** [comment]
Just rebased this and tested. Everything still seems to work.
Anything preventing merge @skjnldsv ?

**Comment by skjnldsv** [comment]
> Just rebased this and tested. Everything still seems to work.
> Anything preventing merge @skjnldsv ?

I forgot about this or 🙈

**Pull request merged**